samples/subsys/usb/dfu/prj.conf

```diff
@@ -3,7 +3,5 @@ CONFIG_GPIO=y
 CONFIG_ARC_INIT=n
 CONFIG_USB=y
 CONFIG_USB_DEVICE_STACK=y
-CONFIG_SYS_LOG_USB_DRIVER_LEVEL=0
```

Comment:
I would like to keep it, have used it regularly.

Reply:
How can you use it? I think the idea of project file is to specify only non-default values, otherwise it looks clumsy.

Reply:
by changing the value :smile:, but yes please set default to 1

Reply:
I suspect it wouldn't work unless we enable CONFIG_SYS_LOG

Reply:
you're right, please put it in